client/src/components/relatedProducts/outfitList.jsx

```diff
@@ -18,48 +18,81 @@ const responsive = {
 };
 
 const Outfit = (props) => {
-  const [productData, setProductData] = React.useState([]);
-  const [list, setList] = React.useState([<AddOutfit key = {0} handleClick = {()=>{console.log('test add button')}}/>])
+  const [productData, setProductData] = React.useState(null);
+  const [input, setInput] = React.useState(<div>test</div>)
+  const [list, setList] = React.useState([])
+
 
   //WILL RECIEVE FROM PROPS
   const product_id = "37314"
-  const product_style = 0
+  const product_style = 221014
 
   React.useEffect(() => {
-
     axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${product_id}`, { headers: { Authorization: API_KEY } })
       .then(response => {
         let productInfoObj = response.data
 
         axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${product_id}/styles`, {
           headers: { Authorization: API_KEY }
         })
+          .catch(err => { console.log('ERROR IN STYLES CALL FOR OUTFIT: ', err) })
           .then(response => {
             let productStylesObj = response.data;
 
             axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/meta?product_id=${product_id}`, {
               headers: { Authorization: API_KEY }
             })
+              .catch(err => { console.log('ERROR IN REVIEW CALL FOR OUTFIT: ', err) })
               .then(response => {
                 let productReviewsObj = response.data
 
                 setProductData({
                   info: productInfoObj,
-                  styles: productStylesObj.results[product_style],
+                  styles: productStylesObj,
                   reviews: productReviewsObj
                 })
               })
+              .catch(err => (console.log('ERROR IN setProductData CALL FOR OUTFIT: ', err)))
           })
       })
-  }, []);
-
-
+  }, [])
 
   return (
     <div>
       <Carousel responsive={responsive}>
-        <AddOutfit handleClick = {()=>{console.log('test add button')}}/>
-        {list.map(card => card)}
+        {
+          [
+            <div key={0}>
```

Comment:
typically a "key" is not a static value but rather something to identify each element and make them unique. Is there a reason you put the key as 0?